Sources/StreamChat/Models/Channel.swift

```diff
@@ -96,6 +96,13 @@ public struct _ChatChannel<ExtraData: ExtraDataTypes> {
     /// and using a `ChatChannelController` for this channel id.
     ///
     public let latestMessages: [_ChatMessage<ExtraData>]
+
+    /// Pinned messages present on the channel.
+    ///
+    /// This field contains only the pinned messages of the channel. You can get all existing messages in the channel by creating
+    /// and using a `ChatChannelController` for this channel id.
+    ///
```

Comment:
Now when we have it, it would make sense to actually use the `@Cached` mechanism for `pinnedMessages`, too.